src/base.ts

```diff
@@ -35,10 +38,16 @@ export class MicrosoftConnector extends BaseHttpConnector<
     ).value
 
     for (const event of Object.values(this.eventConfigurations)) {
-      const options: MicrosoftEventConfigOptions = event.options
+      const { runtimeBaseUrl, ...options } = event.options as MicrosoftConnectorConfigOptions
+      const eventSubscription = {
+        ...options,
+        notificationUrl: runtimeBaseUrl + DEFAULT_WEBHOOK_PATH,
```

Comment:
Is it the DEFAULT_WEBHOOK_PATH here, or do we let the user alter the value?

Reply:
I'll add it as an option